src/app/configuration/line-listing/line-listing-item/line-listing-item.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="line-listing-item-wrapper">
+  <div class="line-listing-btns">
+    <button mat-flat-button color="primary" (click)="openLineListingModal()">
+      {{ 'strings.headingEditLineListing' | translate }}
+    </button>
+
+    <button mat-button color="primary" (click)="disableAllLineListings()">
+      {{ 'captions.lineListingDisableAll' | translate }}
+    </button>
+  </div>
+  <div class="line-listing-content">
+    <span *ngIf="!regionId">
```

Comment:
you can't have block elements inside inline elements
i think it's better to use ng-container instead of span

Reply:
fixed